storage/innobase/xtrabackup/test/t/estimate_memory.sh

```diff
@@ -144,6 +148,15 @@ rm -rf $mysql_datadir
 # PXB-2980 - Add a parameter to disable memory estimation during backup
 start_server
 backupdir=${topdir}/backup
+run_cmd $MYSQL $MYSQL_ARGS test <<EOF
+CREATE TABLE pxb_2710(a INT PRIMARY KEY AUTO_INCREMENT) ENGINE=InnoDB;
+INSERT INTO pxb_2710 VALUES (NULL);
```

Comment:
It is still possible that before 0.5 seconds we reach the ON test. Is one insert sufficient?

Reply:
There is an insert into a loop as part of `run_insert` function. That one is the one that ensures redo log will be generated.

Reply:
But I think it is a cat and mouse game marce. Cat = redo checkpoint thread mouse = pxb :smile:

Reply:
I can remove the sleep, I just did not want to introduce too much unnecessary pressure.

```
run_insert &
insert_pid=$!
```
This will be running in the background and will add 2 redo logs per second. PXB never completes in less than 1 second.

Reply:
Since the test already requiredthe debug PXB binaries. Why not use debug sync to insert a record that would guarnted generated redo before the backup is complete?

Reply:
Can be done that way. Will have the same outcome as the current approach.